docs/css/style.css

```diff
@@ -1262,14 +1262,22 @@ video {
 
 
 /* Media Queries */
-@media (max-width: 992px) {
+@media (max-device-width: 992px) {

```

Comment:
Are you sure this won't break on desktop? Previous rule was applying when browser window is less than 992px wide.